collections/infrastructure/roles/repositories/vars/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-repositories_client_role_version: 1.3.7
+repositories_client_role_version: 1.3.8
 
-repositories_j2_simple_url: "http://{{ networks[j2_node_main_network]['services']['repositories4'][0]['ip4'] | default(networks[j2_node_main_network]['services']['repositories'][0]['ip4'], true) | default(networks[j2_node_main_network]['services_ip'], true) | default('', true) }}/repositories/{{ os_operating_system['repositories_environment'] | default('', true) }}/{{ os_operating_system['distribution'] | default('', true) }}/{{ os_operating_system['distribution_version'] | default(os_operating_system['distribution_major_version']) | default('', true)}}/{{ ansible_architecture }}/"
+repositories_j2_simple_url: "http://{{ networks[repositories_client_network]['services']['repositories4'][0]['ip4'] | default(networks[repositories_client_network]['services']['repositories'][0]['ip4'], true) | default(networks[repositories_client_network]['services_ip'], true) | default('', true) }}/repositories/{{ os_operating_system['repositories_environment'] | default('', true) }}/{{ os_operating_system['distribution'] | default('', true) }}/{{ os_operating_system['distribution_version'] | default(os_operating_system['distribution_major_version']) | default('', true)}}/{{ ansible_architecture }}/"
```

Comment:
Can you rename this variable to `repositories_network` instead ? The role has been renamed.

Also, I would prefer to use default here, instead of setting a j2 in defaults, to avoid losing synchronisation between roles.

Could set

`repositories_client_network:`

In defaults, and here:

`networks[repositories_client_network | default(j2_node_main_network, true)]`

?

Both way works, just to ensure all roles work the same way :slightly_smiling_face: 

Can you also add a small ntoe in the README file about this new variable ? Just 2 lines, to expose it.

Reply:
Thanks for the review. Will apply the requested changes